deploy/internal/service-s3.yaml

```diff
@@ -4,6 +4,7 @@ metadata:
   name: s3
   labels:
     app: noobaa
+    noobaa-s3: "true"
```

Comment:
same. maybe change to `noobaa-s3-svc: "true"`

Reply:
if you change the labels don't forget to change the selectors using them

Reply:
done